src/components/Folder.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import styled, { css } from 'styled-components/native'
+import { type Note } from '../types'
+
+const Text = styled.Text`
+  font-size: 14px;
+  color: ${props => props.theme.white};
+  margin-left: 10px;
+  margin-bottom: 5px;
+  cursor: pointer;
+  display: block;
+`
+
+const FolderContainer = styled.View`
+  display: inline;
+`
+
+const CollapseFolder = styled.Text`
+  color: ${props => props.theme.white};
+  font-weight: bold;
+  cursor: pointer;
+`
+
+const FolderText = styled.TextInput`
+  font-size: 15px;
+  color: ${props => props.theme.white};
+  margin-bottom: 5px;
+  margin-top: 10px;
+  cursor: pointer;
+`
+
+const FolderFlatList = styled.FlatList`
+  display: block;
+  ${props =>
+    props.open !== -1 &&
+    css`
+      display: none;
+    `}
+`
+type Props = {
+  data: Array<Note>,
+  folderID: string,
+  folderName: string,
+  folderDraggable: boolean,
+  open: number,
```

Comment:
Could `open` be a boolean instead? Or if not bool, maybe a new type with its possible states enumerated?

Also, would `isOpen` be a better name?